tests.js

```diff
@@ -170,5 +170,72 @@ describe('slugifyId()', function () {
   it('should do nothing if the id is undefined', function () {
     expect(kibiutils.slugifyId(undefined)).to.be(undefined);
   });
+});
+
+describe('doesQueryDependOnEntity()', function () {
+  var notDependentQueries = [
+      'select * \n' +
+       'from test \n' +
+       '-- where name = \'@doc[_source][github_id]@\''
+    ,
+      'select * \n' +
+       'from test \n' +
+        '/* where name \n' + '= \'@doc[_source][github_id]@\' */'
+    ,
+      'select * \n' +
+       'from test \n' +
+       '# where name = \'@doc[_source][github_id]@\''
+    ,
+      'select * \n' +
+       'from test \n' +
+        '/* where name \n' + '= \'@doc[_source] \n ' + '[github_id]@\ */'
+  ];
+
+  var DependentQueries = [
+     'select * \n' +
+       'from test \n' +
+       ' where name = \'@doc[_source][github_id]@\''
+    ,
+      'select * \n' +
+       'from test \n' +
+        '- where name = \'@doc[_source][github_id]@\''
+    ,
+      'select * \n' +
```

Comment:
why is this query dependent ? The block comment doesn't finish.

The query seems incomplete. I think it should be something like:

```sql
select *
from test
/* where name
= 123 */ where name = '@doc[_source][github_id]@'
```